File: Java/1012 - Área.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        double a, b, c;
        double triangulo, circulo, trapezio, quadrado, retangulo;
        double areaTriangulo, areaCirculo, areaTrapezio, areaQuadrado, areaRetangulo;
    
        Scanner valor = new Scanner(System.in);
        
        a = valor.nextDouble();
        b = valor.nextDouble();
        c = valor.nextDouble();
        
        areaTriangulo = (a * c) / 2.0;

        areaCirculo = (c * c) * 3.14159;
        
        areaTrapezio = ((a + b) * c) / 2;
        
        areaQuadrado = b * b;
        
        areaRetangulo = a * b;
        
        System.out.println(String.format("TRIANGULO: %.3f", areaTriangulo));
        System.out.println(String.format("CIRCULO: %.3f", areaCirculo));
        System.out.println(String.format("TRAPEZIO: %.3f", areaTrapezio));
        System.out.println(String.format("QUADRADO: %.3f", areaQuadrado));
        System.out.println(String.format("RETANGULO: %.3f", areaRetangulo));
    }
}

File: C#/1012 - Área.cs
using System; 
  
 namespace Area{ 
     class Program{ 
        static void Main(string[] args){ 
            double areaTriangulo, areaCirculo, areaTrapezio, areaQuadrado, areaRetangulo;
            
            // Recebe valores
            String valor = Console.ReadLine();
            string[] medida = valor.Split(' ');
            double a = Convert.ToDouble(medida[0]);
            double b = Convert.ToDouble(medida[1]);
            double c = Convert.ToDouble(medida[2]);
            
            // Calculo
            areaTriangulo = (a * c) / 2.0; 
            areaCirculo = Math.Pow(c, 2) * 3.14159; 
            areaTrapezio = ((a + b) * c) / 2.0; 
            areaQuadrado = Math.Pow(b, 2); 
            areaRetangulo = a * b;

            Console.WriteLine(string.Format($"TRIANGULO: {areaTriangulo.ToString("0.000")}")); 
            Console.WriteLine(string.Format($"CIRCULO: {areaCirculo.ToString("0.000")}")); 
            Console.WriteLine(string.Format($"TRAPEZIO: {areaTrapezio.ToString("0.000")}")); 
            Console.WriteLine(string.Format($"QUADRADO: {areaQuadrado.ToString("0.000")}")); 
            Console.WriteLine(string.Format($"RETANGULO: {areaRetangulo.ToString("0.000")}")); 
        }
    }
}
